src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs

```diff
@@ -18,6 +18,8 @@ public sealed partial class JsonSerializerOptions
         // The global list of built-in simple converters.
         private static readonly Dictionary<Type, JsonConverter> s_defaultSimpleConverters = GetDefaultSimpleConverters();
 
+        private static readonly JsonNodeConverterFactory s_JsonNodeConverterFactory = new JsonNodeConverterFactory();
```

Comment:
Why is this singled out and cached?

Reply:
Probably to minimize the allocations discussed in https://github.com/dotnet/runtime/pull/51025#discussion_r612833589

Reply:
Combined now. This is a remnant from a previous iteration that needed to dynamically create the converter to avoid the reference to System.Linq.Expressions.dll